README.md

```diff
@@ -69,23 +78,37 @@ driver.findElementByAccessibilityId("login").sendKeys('Hello');
 ```
 ## Configure Wait timeout in test
 
+While overriding timeouts one can choose to override all or one of the below
+* timeout
+* intervalBetweenAttempts
+
 WDIO Example 
 
 ```
-await driver.executeScript('plugin: setWaitTimeout', [
+await driver.executeScript('plugin: setWaitPluginProperties', [
```

Comment:
Can it be just setWait? Signature or contract gives us optional properties isn't it?

Reply:
Same for getWait as well. Most of command in Appium does takes certain properties so I don't think we need to call out explicitly.

Reply:
@SrinivasanTarget It needs to be `setWaitProperties` or `setWaitPluginProperties ` just giving setWait meaning this property not only sets timeout options but also more.